packages/ember-application/lib/system/application.js

```diff
@@ -316,46 +295,19 @@ var Application = Namespace.extend(DeferredMixin, {
     Create a container for the current application's registry.
 
     @private
-    @method buildContainer
+    @method buildInstance
     @return {Ember.Container} the configured container
   */
-  buildContainer: function() {
+  buildInstance: function() {
     var container = this.__container__ = this.__registry__.container();
 
-    return container;
-  },
-
-  /**
-    If the application has not opted out of routing and has not explicitly
-    defined a router, supply a default router for the application author
-    to configure.
-
-    This allows application developers to do:
-
-    ```javascript
-    var App = Ember.Application.create();
-
-    App.Router.map(function() {
-      this.resource('posts');
+    var instance = this.__instance__ = ApplicationInstance.create({
```

Comment:
Can you say more what you mean? The `ApplicationInstance` is responsible for creating the container.